pages/review/protectedApplication.tsx

```diff
@@ -0,0 +1,79 @@
+import { ParsedUrlQuery } from "querystring";
+import { ReactChild, ReactElement, useEffect, useState } from "react";
+import jwt_decode from "jwt-decode";
+import { fetchGraphql } from "@utils/makegqlrequest";
+import { queries } from "graphql/queries";
+import Loading from "@components/common/Loading";
+
+type Props = {
+  children: ReactChild;
+  headerInformation: ParsedUrlQuery;
+};
+
+type AccessToken = {
+  readonly user_id: string;
+};
+
+const getReviewId = (query: ParsedUrlQuery): number => {
+  // verify reviewId
+  const reviewId =
+    typeof query["reviewId"] === "string"
+      ? parseInt(query["reviewId"])
+      : (() => {
+          throw new Error("reviewId must be a String");
+        })();
+  if (Number.isNaN(reviewId))
```

Comment:
someone once docked points from me in an interview for not using braces in my if statements, even if they were only one line.  the idea was that
```
if (Number.isNaN(reviewId))
  doSomething()
  throw Error()
```
is very easy to let slip by in code review and should be avoided, since it doesn't do what is expected.

i don't really care in this situation lol but I thought it might be helpful to know that people look for it in interviews!

Reply:
Yep, let me add some error handling and style changes to the above